.github/workflows/swift.yml

```diff
@@ -4,13 +4,18 @@ on: [push, pull_request]
 
 jobs:
   build:
-    runs-on: macos-latest
+    runs-on: macos-11.0
     
     steps:
     - uses: actions/checkout@v2
     - uses: fwal/setup-swift@v1
       with:
         swift-version: "5.3"
```

Comment:
Is this necessary or is it possible to avoid using it thereby using the default swift version for the xcode version in question?

Reply:
Swift 5.3 is the default provided by fwal/setup-swift, we could omit the swift-version requirement.